ui/dependencies/_typography.scss

```diff
@@ -41,3 +41,8 @@
   text-transform: uppercase;
   letter-spacing: 0.0625rem;
 }
+
+// Added for Picsa OS for better display of Proxima Nova
```

Comment:
@stiaan does this act as a type of placeholder comment for future changes..?  If not, you can remove